src/Temporalio/Worker/WorkflowInstance.cs

```diff
@@ -1292,7 +1309,7 @@ await inbound.Value.HandleSignalAsync(new(
             }));
         }
 
-        private void ApplyStartWorkflow(StartWorkflow start)
+        private void ApplyStartWorkflow(InitializeWorkflow init)
```

Comment:
```suggestion
        private void ApplyInitializeWorkflow(InitializeWorkflow init)
```

Pedantic, don't have to change